install_debs_into_image.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# customize raspbian image for revolution pi
+
+usage () {
+	echo 'Usage: customize_image.sh [-m, --minimize | -h, --help] <image>
+  -m, --minimize	Install only software that is necessary for basic operation (eg. Pictory and other RevPi tools)
+  -h, --help		Print the usage page'
+}
+
+if [ ! -x "$(which curl)" ]; then
+	echo 1>&1 "Error: Command curl not found."
+	exit 1
+fi
+
+if [ ! -x "$(which fsck.vfat)" ]; then
+	echo 1>&1 "Error: Command fsck.vfat not found."
+	exit 1
+fi
+
+if [ ! -x "$(which lsof)" ]; then
+	echo 1>&1 "Error: Command lsof not found."
+	exit 1
+fi
+
+PARTED="$(which parted)"
+if [ "x$PARTED" = "x" ] ; then
+	echo 1>&1 "Error: Command parted not found."
+	exit 1
+fi
+
+if [ ! -x "$PARTED" ] ; then
+	echo 1>&1 "Error: Command $PARTED is not executable."
+	exit 1
+fi
+
+set -ex
+
+# pivot to new PID namespace
+if [ $$ != 2 ] && [ -x /usr/bin/newpid ] ; then
+	exec /usr/bin/newpid "$0" "$@"
+fi
+
+# set MINIMG as 0: build the normal image by default
+MINIMG=0
+
+# get the options
+if ! MYOPTS=$(getopt -o mh --long minimize,help -- "$@"); then
+	usage;
+	exit 1;
+fi
+eval set -- "$MYOPTS"
+
+# extract options and their arguments into variables.
+while true ; do
+	case "$1" in
+		-m|--minimize) MINIMG=1 ; shift ;;
```

Comment:
I think we can omit the whole minimg handling in this script, as it is not used at all.

Reply:
good idea